extensions/tmdb/package.json

```
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "tmdb",
+  "title": "TMDB",
+  "description": "Browse through hundreds of thousands of movies and tv shows, see new releases, parental guide and much more",
+  "icon": "command-icon.png",
+  "author": "Aayush9029",
+  "categories": [
+    "Media",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "searchMovie",
+      "title": "Search Movie",
+      "subtitle": "Search for a movie by name",
```

Whatever you change the title of the extension to, I would add it as subtitle in the commands, makes it easier to look for them in Raycast
```suggestion
      "subtitle": "The Movie Database",
```